modules/page_base.py

```diff
@@ -478,6 +532,11 @@ def wait_for_num_tabs(self, num_tabs: int) -> Page:
             logging.warn("Timeout waiting for the number of windows to be:", num_tabs)
         return self
 
+    def switch_tab(self):
```

Comment:
I think this method has a misleading name, perhaps `switch_to_new_tab` is a better name?